sdks/python/apache_beam/io/hdfs_integration_test/Dockerfile

```diff
@@ -0,0 +1,38 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# This image contains a Python SDK build and dependencies.
+# By default it runs wordcount against a locally accessible HDFS service.
+# See hdfs_integration_test.sh for example usage.
+
+FROM python:2
+WORKDIR /app
+RUN pip install --no-cache-dir holdup
+
+# Install Beam and dependencies.
+ADD sdks/python /app/sdks/python
```

Comment:
Do we need/want to run this python test in a container? It seems we can run the test the same way we're running other python tests, now that we're using a python HDFS client.

If this is the structure we want, an option might be to map the needed directories into a fixed "python test" image as host volumes instead of ADDing them to a new image just to run them.

Reply:
I find putting everything in a Docker network makes it easier to hook up the namenode, datanode, and test using static port numbers. Correct me if I'm wrong, but I believe the other tests launch local services.